static-site/src/components/splash/index.jsx

```diff
@@ -211,61 +213,179 @@ export default Splash;
 /*** SHOWCASE ***/
 
 const UrlShowcaseTile = ({ card }) => {
+
+  /* Narrative detection works for all three sources:
+  1. Core: /narratives/<narrative path>
+  2. Community: /community/narratives/<owner>/<narrative path>
+  3. Groups: /groups/<group>/narratives/<narrative path>
+
+  Including slashes in the check prevents false positives.
+  */
+  const isNarrative = card.url.includes('/narratives/');
+
   return (
     <CardOuter>
       <CardInner>
         <a href={card.url}>
-          <CardTitle $squashed>
+          <CardTitle>
             {card.name}
           </CardTitle>
-          <CardImgWrapper filename={card.img}/>
+          <CardImgContainer>
+            <CardImgWrapper filename={card.img}/>
+            <InfoIcons>
+              <CardSourceIcon url={card.url} isNarrative />
```

Comment:
The prop here needs to be changed to `isNarrative={isNarrative}` - the current syntax is a [default true prop](https://legacy.reactjs.org/docs/jsx-in-depth.html#props-default-to-true), which is why it passes linting, but it's not behaving like it does when declaring objects. This has the flow on effect (within `CardSourceIcon`) of returning the wrong GitHub org name because everything's a narrative!